### PR TITLE
feat(csv): load csv shipments and orders

### PR DESCRIPTION
**Pull request by nahuelRo**

### CODE REVIEW

File: api/models/Shipment.models.ts [hunk 1]
```diff
@@ -1,40 +1,38 @@
...
+      type: String,
+      required: true,
+    },
+    price: {
+      type: String,
+      required: true,
+    },
+    status: {
+      type: String,
+      enum: ["Missing Data", "Delivered", "Preparing", "Avaliable", "Shipped"],
```
Comment (sebikap-gm2) [inline api/models/Shipment.models.ts]: Use array from types

**Review by sebikap-gm2 — CHANGES_REQUESTED** [review]

File: api/validations/order.validation.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import { ORDER_STATUSES } from "../types";
+import { isISODate } from "../utils";
+import { z } from "zod";
+
+export const OrderValidation = z.object({
+  member: z.string().min(1),
```
Comment (sebikap-gm2) [inline api/validations/order.validation.ts:7]: Insert entire member

File: api/models/Order.models.ts [hunk 3]
```diff
@@ -1,33 +1,27 @@
...
-  ],
-});
-
-export const OrderRepository = mongoose.model("Orders", OrdersSchema);
+import { ORDER_STATUSES } from "../types";
+import mongoose from "mongoose";
+
+const OrdersSchema = new mongoose.Schema(
+  {
+    member: {
```
Comment (sebikap-gm2) [inline api/models/Order.models.ts:7]: Should be entire member

**Review by sebikap-gm2 — CHANGES_REQUESTED** [review]

File: api/validations/order.validation.ts (hunk 2 above)
Comment (nahuelRo) [reply api/validations/order.validation.ts:7]: Do not insert the member, as the member's first and last name is still requested within the order and shipments.

Comment (sebikap-gm2) [reply api/validations/order.validation.ts:7]: Add //TODO

**Review by sebikap-gm2 — APPROVED** [review]

**Pull request merged**